eci-eks-dev/k8s-resources/eciflow/00-namespace/eciflow-namespace.yaml

```
@@ -0,0 +1,4 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: eciflow
```

I thinks its better to have labels in metadata similar to AWS tags ( not sure about the key/values, I just added example here as suggestion ). We will have some standardized way of labeling in the future. 

```suggestion
  name: eciflow
  labels:
    product_domain: ent 
    team: eci
```